frontend/assets/javascripts/src/modules/form/paypal.es6

```diff
@@ -32,14 +32,21 @@ function setupPayment (resolve, reject) {
 
 	if (payment.validateForm()) {
 
+        const checkoutForm = guardian.membership.checkoutForm;
+        const amount = checkoutForm.billingPeriods[checkoutForm.billingPeriod].amount[checkoutForm.currency];
```

Comment:
This is true, the user needs to select a country before they can attempt payment or they get a regular form validation error